MODULE.bazel

```diff
@@ -0,0 +1,35 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+module(
+    name = "skywalking-data-collect-protocol",
+    version = "10.0.0",
```

Comment:
We have minor changes with versions, I am afraid we don't update this from version to version, because we don't use Brazil internally.

Reply:
We don't have this release plan at there.

Reply:
I will ask our maintainer of cpp SDK to take a look.

Reply:
@wbpcode Could you check on this? cpp2sky is the only one using this tool(Brazel).